src/main/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleGroupSyncMessageReceiver.scala

```diff
@@ -67,6 +67,8 @@ class GoogleGroupSyncMessageReceiver(groupSynchronizer: GoogleGroupSynchronizer)
     import WorkbenchIdentityJsonSupport._
     import org.broadinstitute.dsde.workbench.sam.google.SamGoogleModelJsonSupport._
 
+    val foo = report.map { case (group, syncReportItems) => (group, syncReportItems.groupBy(_.operation)) }
```

Comment:
is this `foo` for testing?

Reply:
yup! Good catch!

Reply:
I'm honestly surprised the compiler didn't yell at me for an unused variable.